inc/az_ulib_ipc_interface.h

```diff
@@ -71,123 +79,6 @@ typedef struct
- */
-AZ_INLINE AZ_NODISCARD az_result azi_ulib_ipc_try_get_interface(
-    const az_ulib_ipc_table* const table,
-    az_span name,
-    az_ulib_version version,
-    az_ulib_version_match_criteria match_criteria,
-    az_ulib_ipc_interface_handle* interface_handle)
-{
-  return table->try_get_interface(name, version, match_criteria, interface_handle);
-}
-
-/*
- * @brief   Dynamically linked wrapper to az_ulib_ipc_try_get_capability().
- */
-AZ_INLINE AZ_NODISCARD az_result azi_ulib_ipc_try_get_capability(
-    const az_ulib_ipc_table* const table,
-    az_ulib_ipc_interface_handle interface_handle,
-    az_span name,
-    az_ulib_capability_index* capability_index)
-{
-  return table->try_get_capability(interface_handle, name, capability_index);
-}
-
-/*
- * @brief   Dynamically linked wrapper to az_ulib_ipc_get_interface().
- */
-AZ_INLINE AZ_NODISCARD az_result azi_ulib_ipc_get_interface(
-    const az_ulib_ipc_table* const table,
-    az_ulib_ipc_interface_handle original_interface_handle,
-    az_ulib_ipc_interface_handle* interface_handle)
-{
-  return table->get_interface(original_interface_handle, interface_handle);
-}
-
-/*
- * @brief   Dynamically linked wrapper to az_ulib_ipc_release_interface().
- */
-AZ_INLINE AZ_NODISCARD az_result azi_ulib_ipc_release_interface(
-    const az_ulib_ipc_table* table,
-    az_ulib_ipc_interface_handle interface_handle)
-{
-  return table->release_interface(interface_handle);
-}
-
-/*
- * @brief   Dynamically linked wrapper to az_ulib_ipc_call().
- */
-AZ_INLINE AZ_NODISCARD az_result azi_ulib_ipc_call(
-    const az_ulib_ipc_table* const table,
-    az_ulib_ipc_interface_handle interface_handle,
-    az_ulib_capability_index capability_index,
-    az_ulib_model_in model_in,
-    az_ulib_model_out model_out)
-{
-  return table->call(interface_handle, capability_index, model_in, model_out);
-}
-
-/*
- * @brief   Dynamically linked wrapper to az_ulib_ipc_call_with_str().
- */
-AZ_INLINE AZ_NODISCARD az_result azi_ulib_ipc_call_with_str(
-    const az_ulib_ipc_table* const table,
-    az_ulib_ipc_interface_handle interface_handle,
-    az_ulib_capability_index capability_index,
-    az_span model_in_span,
-    az_span* model_out_span)
-{
-  return table->call_with_str(interface_handle, capability_index, model_in_span, model_out_span);
-}
-
-/*
- * @brief   Dynamically linked wrapper to az_ulib_ipc_query().
- */
-AZ_INLINE AZ_NODISCARD az_result azi_ulib_ipc_query(
-    const az_ulib_ipc_table* const table,
-    az_span query,
-    az_span* result,
-    uint32_t* continuation_token)
-{
-  return table->query(query, result, continuation_token);
-}
-
-/*
- * @brief   Dynamically linked wrapper to az_ulib_ipc_query_next().
- */
-AZ_INLINE AZ_NODISCARD az_result azi_ulib_ipc_query_next(
-    const az_ulib_ipc_table* const table,
-    uint32_t* continuation_token,
-    az_span* result)
-{
-  return table->query_next(continuation_token, result);
-}
-
```

Comment:
how are people supposed to call the table APIs without these?

Reply:
Calling the pointer directly. For example
```
az_result result = table->publish(interface_descriptor, interface_handle);
```